BFieldGeom/inc/BFMapType.hh

```diff
@@ -27,11 +27,11 @@ namespace mu2e {
 
     class BFMapType {
        public:
-        enum enum_type { unknown, GMC, G4BL, PARAM, lastEnum };
+        enum enum_type { unknown, unused, G4BL, PARAM, lastEnum };
```

Comment:
This class is not persisted in any data product.  So it is safe to remove unused entries instead of renaming them to unused.

Reply:
Done